pepper-apis/docs/specification/src/pepper.yml

```diff
@@ -6,7 +6,7 @@ servers:
     description: Production server (uses real data)
 info:
   description: 'Pepper API specification'
-  version: '1.16.1-863fa496'
+  version: 'v1.25-46-gcd81179ef'
```

Comment:
is the version a random string?

Reply:
Nope. There isn't an overarching versioning scheme for the API, so this is the output from a git describe call prior to the commit. It's not precise, but close enough for our needs.